Automattic/wp-calypso
# Reader: move email settings popover to new block

So that we can reuse the Email Settings popover functionality elsewhere (specifically the stream - see https://github.com/Automattic/wp-calypso/issues/15457), this PR relocates the component responsible from inside `reader-subscription-list-item` to a new block, `reader-email-settings`.

Styles have been migrated and classes renamed. I've also added a couple of missing translated strings.

### To test 

Ensure that there are no visual or functional regressions in the email settings popover on http://calypso.localhost:3000/following/manage.

You can also check out the new Devdocs page at:

http://calypso.localhost:3000/devdocs/blocks/reader-email-settings
